### PR TITLE
Allow including hashes of method signatures in --combined-json output

### PR DESCRIPTION
**Pull request by ruchevits**

### CODE REVIEW

File: libsolidity/interface/CompilerStack.cpp [hunk 1]
```diff
@@ -481,6 +481,14 @@ Json::Value const& CompilerStack::natspec(Contract const& _contract, Documentati
 	return *(*doc);
 }
 
+Json::Value CompilerStack::functionHashes(ContractDefinition const& _contract)
```
Comment (chriseth) [inline libsolidity/interface/CompilerStack.cpp:484]: Can you please remove the respective function from jsonCompiler.cpp?

Comment (ruchevits) [reply libsolidity/interface/CompilerStack.cpp:484]: Done

Comment (chriseth) [reply libsolidity/interface/CompilerStack.cpp:484]: Hm, the other version seems to have had a mapping signature -> hex - could you change that please?

Comment (chriseth) [reply libsolidity/interface/CompilerStack.cpp:484]: I mean change it so that the old behaviour does not change.

**Comment by axic** [comment]
There is also a copy of this method in `interface/StandardCompiler.cpp:methodIdentifiers`. I'd prefer calling it internally `methodIdentifiers` as that is the version we've chosen for the standard JSON IO.

**Review by chriseth — APPROVED** [review]
The current state of the PR is an improvement, so I will merge it.

**Pull request merged**